### ISSUE TITLE
Map Server 5.6  - Map filter issue with oracle spatial data

### ISSUE THREAD

**Issue by Roger1603** [mapserver/mapserver#4339]
Hi 
Recently , we have upgrade map server from v4.6 to V5.6. This application work fine on older map version 4.6 which filter oracle spatial data on the fly and it render map nicely. 
However ,after the map upgrade with v5.6 that  application does not filter that layer. it render all the data on the layer. 
I use oracle SQL web tool to create application. 

Here is the code that use to in that program.
vFilters  - this will get selected record No's to filter

and the it passes it values on to  map CGI variable "layer" 
<input type="hidden" name="layers" value="'|| vFilters  > 

Then along with other MAP CGI variable it submits to map server execute that request.

Can any one help me out to sort this issue .

Thank you 
Roger

**Comment by jratike80** [comment]
It is hard to follow how the PL/SQL code is sending and what as variables to be used. It feels like this is not a bug but a question about how to use variables, variable substitution, and validation with PL/SQL and Mapserver. Reopen if you can show with shp2img or WMS GetMap request that runtime substitution and validation system has a bug.
